### ISSUE TITLE
Command failed: /usr/bin/git checkout conf-override

### ISSUE THREAD

**Issue by sheldonzf** [tapjs/tapjs#539]
When i install tap@13.0.0，it will report error.
```
npm ERR! Command failed: /usr/bin/git checkout conf-override
npm ERR! error: pathspec 'conf-override' did not match any file(s) known to git.
npm ERR!
```

**Comment by trygve-lie** [comment]
I am getting the same both locally and on Travis.

Examples:
https://travis-ci.org/trygve-lie/ttl-mem-cache/builds/524731891
https://travis-ci.org/metrics-js/process/builds/524731002
https://travis-ci.org/podium-lib/fastify-podlet/builds/524731287

**Comment by trygve-lie** [comment]
Here is some more info from the npm log;

```sh
1671 silly pacote range manifest for yapool@^1.0.0 fetched in 6ms
1672 silly resolveWithNewModule yapool@1.0.0 checking installable status
1673 silly fetchPackageMetaData error for nyc@github:isaacs/nyc#conf-override Command failed: git checkout conf-override
1673 silly fetchPackageMetaData error: pathspec 'conf-override' did not match any file(s) known to git.
1674 timing stage:rollbackFailedOptional Completed in 1ms
1675 timing stage:runTopLevelLifecycles Completed in 10809ms
1676 silly saveTree abslog@2.4.0
1676 silly saveTree ├─┬ bunyan@1.8.12
1676 silly saveTree │ ├─┬ dtrace-provider@0.8.7
```

**Comment by DeannaLou** [comment]
+1

**Comment by anlin-yang** [comment]
I get the same problem. Nothing changes。

**Comment by delvedor** [comment]
Hello! Same error here.

**Comment by goto-bus-stop** [comment]
Looks like a dependency change that was supposed to be temporary leaked into the release, #540 fixes the problem.

**Comment by NatureFeng** [comment]
Same error.

**Comment by isaacs** [comment]
Oooops.  Sorry about that.  Fixed  by #540.
